src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/LinearAVF.java

```diff
@@ -337,7 +212,7 @@ public LinearAVF setWeight(Criterion awt, double value) {
    * @param value >= 0
    */
   private void setFloorAreaSubjectiveValueWeight(double value) {
```

Comment:
Dans la javadoc il est indiqué que la valeur "value" doit être supérieure ou égale à 0. 
Si cela est effectivement le cas, il faut ajouter un checkArgument, sinon il faut modifier la javadoc (ici et dans tous les setCriterionSubjectiveValueWeight().

Reply:
Fix in d475cb0bc1a476a05cfd0eb89a9d9013da5b7cda